ui/app/controllers/vault/cluster/clients.ts

```diff
@@ -3,11 +3,14 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Component from '@glimmer/component';
+import Controller from '@ember/controller';
 import { service } from '@ember/service';
-
 import type VersionService from 'vault/services/version';
 
-export default class SyncLandingCtaComponent extends Component {
```

Comment:
Really weird git diff decision here - `landing-cta.ts` was actually deleted ❌  and `cluster/clients.js` above was converted to typescript